src/UpdateCheck.lua

```diff
@@ -216,19 +235,7 @@ for index, data in ipairs(updateFiles) do
 		end
 	else
 		ConPrintf("Downloading %s... (%d of %d)", data.name, index, #updateFiles)
-		content = downloadFileText(source, data.name)
-	end
-	if content then
-		if data.sha1 ~= sha1(content) and data.sha1 ~= sha1(content:gsub("\n","\r\n")) then
```

Comment:
I was a bit nervous about removing this at first, but I'm not sure what the original purpose was, other than maybe a sanity check for if the manifest was updated then nothing should be missing?